sunpy/map/compositemap.py

```diff
@@ -434,6 +434,11 @@ def plot(self, axes=None, annotate=True,  # pylint: disable=W0613
             # rendered using imshow.
             if m.levels is False:
                 # Check for the presence of masked map data
+
+                # Check if the linewidths argument is provided, if so, then delete it from params.
+                if matplot_args.get('linewidths', -1) != -1:
+                    del params['linewidths']
```

Comment:
I don't understand how this works. I don't really understand how this plot function works.

Reply:
So, here I am checking if the 'linewidths' key is present matplot_args dictionary, if so then we need to delete that key value pair from the params dictionary which will be provided to imshow function. This is necessary because the imshow will throw an error as it doesn't take 'linewidths' argument.